src/Core/src/Platform/iOS/SearchBarExtensions.cs

```diff
@@ -21,7 +21,7 @@ internal static void UpdateBackground(this UISearchBar uiSearchBar, ISearchBar s
 			var background = searchBar.Background;
 
 			if (background is SolidPaint solidPaint)
-				uiSearchBar.BarTintColor = solidPaint.Color.ToPlatform();
+				ViewExtensions.UpdateBackground(uiSearchBar, solidPaint);
```

Comment:
I'm thinking we might want to split this change out from this PR in order to fix how the background is applied.

If you compare android to ios the SearchBar on iOS is more of a tint
![image](https://user-images.githubusercontent.com/5375137/222851349-3f472bb4-8d6b-4173-a311-322a0d254243.png)

```XAML
 <VerticalStackLayout>
        <SearchBar Background="Purple"></SearchBar>
        <SearchBar>
            <SearchBar.Background>
                <LinearGradientBrush EndPoint="1,0">
                    <GradientStop Color="Yellow" 
                                      Offset="0.1" />
                    <GradientStop Color="Green"                                   
                                      Offset="1.0" />
                </LinearGradientBrush>
            </SearchBar.Background>
        </SearchBar>
        <SearchBar></SearchBar>
        
    </VerticalStackLayout>
</ContentPage>
```

I think to get the searchbar background to match you have to set the background color on one of the inner controls
https://stackoverflow.com/a/48204512

Reply:
Ok, thanks for the comment.  For now I'm undoing that change and leaving the test disabled, with a link to new issue https://github.com/dotnet/maui/issues/13693